gcc/rust/typecheck/rust-tyty-call.cc

```diff
@@ -23,6 +23,35 @@
 namespace Rust {
 namespace TyTy {
 
+void
+emit_unexpected_argument_error (Location loc,
+				unsigned long unexpected_arg_count,
+				unsigned long expected_arg_count)
+{
+  // https://doc.rust-lang.org/error_codes/E0061.html
+  // rustc treats 1 as singular and others as plural
+  std::string err_msg = "this function takes %lu ";
```

Comment:
We do not need to have the same exact error messages as rustc. As long as the error code is the same, it's fine if the error message is a little different as long as it has the same meaning. This isn't an issue, just a note :)

Reply:
Okay :)